test/test_examples.jl

```diff
@@ -1,11 +1,13 @@
 ENV["EMX_TEST"] = true # Set flag for example scripts to check if they are run as part of the tests
 
 @testset "Run examples" begin
-    exdir = joinpath(@__DIR__, "../examples")
+    exdir = joinpath(@__DIR__, "..", "examples")
     files = filter(endswith(".jl"), readdir(exdir))
     for file ∈ files
         @testset "Example $file" begin
-            include(joinpath(exdir, file))
+            redirect_stdio(stdout=devnull, stderr=devnull) do
+                include(joinpath(exdir, file))
+            end
             @test termination_status(m) == MOI.OPTIMAL
```

Comment:
Consider resetting m between test cases to avoid failing examples to be treated as ok if  previous termination status was MOI.OPTIMAL?
```suggestion
            @test termination_status(m) == MOI.OPTIMAL
            m = Model() # Reset m before next example file
```